css/style.css

```diff
@@ -117,9 +119,9 @@ body {
 }
 
 .f-section {
-	height: 95vh;
+	height: 88vh;
 	background: linear-gradient(to left, #fff 50%, #000 50%);
-	background-size: 62.7vw;
+	background-size: 64.6vw;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/LOCO_A/pullRequest?prid=6206892)